tango_simlib/tango_sim_generator.py

```diff
@@ -271,6 +275,13 @@ def _reset_to_default_state(self):
                         adjustable_val = 0.0
                     setattr(simulated_quantity, attr, adjustable_val)
 
+        def fill_new_device_properties(self):
+            """Fill device properties into tangoDB"""
+            db_instance = Database()
+            for prop_name, prop_meta in self.model.sim_properties.items():
+                db_instance.put_device_property(
+                    self.get_name(), {prop_name: prop_meta['DefaultPropValue']})
+
```

Comment:
Are you sure there are no other meta_data on Properties that should be written to the DB?